packages/grid/_modules_/grid/locales/ukUA.ts

```diff
@@ -22,23 +35,22 @@ const ukUAGrid: Partial<GridLocaleText> = {
   // Filters toolbar button text
   toolbarFilters: 'Фільтри',
   toolbarFiltersLabel: 'Показати фільтри',
-  toolbarFiltersTooltipHide: 'Сховати фільтри',
+  toolbarFiltersTooltipHide: 'Приховати фільтри',
   toolbarFiltersTooltipShow: 'Показати фільтри',
-  toolbarFiltersTooltipActive: (count) =>
-    count !== 1 ? `${count} активні фільтри` : `${count} активний фільтр`,
+  toolbarFiltersTooltipActive: getPluralFormForFilter,
 
   // Export selector toolbar button text
   toolbarExport: 'Експорт',
   toolbarExportLabel: 'Експорт',
   toolbarExportCSV: 'Завантажити у форматі CSV',
-  // toolbarExportPrint: 'Print',
+  toolbarExportPrint: 'Друк',
 
   // Columns panel text
   columnsPanelTextFieldLabel: 'Знайти стовпець',
   columnsPanelTextFieldPlaceholder: 'Заголовок стовпця',
   columnsPanelDragIconLabel: 'Змінити порядок стовпця',
-  columnsPanelShowAllButton: 'Показати усі',
-  columnsPanelHideAllButton: 'Сховати усі',
+  columnsPanelShowAllButton: 'Показати все',
```

Comment:
```suggestion
  columnsPanelShowAllButton: 'Показати всі',
```
Показати всі (стовпці)